yt_dlp/extractor/churchofjesuschrist.py

```diff
@@ -0,0 +1,31 @@
+from .common import InfoExtractor
+
+
+class ChurchOfJesusChristIE(InfoExtractor):
+    _VALID_URL = r'https?:\/\/(?:www\.)?churchofjesuschrist\.org\/broadcasts\/watch\/(?P<id>(?:\w|-)*?)\/(?:.*)'
+    _TESTS = [{
+        'url': 'https://www.churchofjesuschrist.org/broadcasts/watch/lethbridge-alberta-member-devotional/2022/10?lang=eng',
+        'md5': '424b7a63b9c3c706d1b1309c3bda90d6',
+        'add_ie': ['BrightcoveNew'],
+        'info_dict': {
+            'id': '6313308990112',
+            'ext': 'mp4',
+            'upload_date': '20221005',
+            'uploader_id': '710857117001',
+            'timestamp': 1664994416,
+            'title': str,
```

Comment:
give full title in test

Reply:
done in https://github.com/yt-dlp/yt-dlp/pull/5409/commits/989ec1aee9bfb21989329554aaa1dcfc98b8b370